test/Kestrel.Core.Tests/HttpParserTests.cs

```diff
@@ -26,8 +26,10 @@ public class HttpParserTests
             string expectedMethod,
             string expectedRawTarget,
             string expectedRawPath,
+#pragma warning disable xUnit1026 // Theory methods should use all of their parameters
```

Comment:
Comment that it's sharing theory data with FrameTests.TakeStartLineSetsFrameProperties.